_includes/menubar.html

```diff
@@ -15,27 +15,33 @@
           <span class="input-group-text bg-white border-right-0 text-muted" id="search-addon">
             <i class="fas fa-search"></i>
           </span>
-          <input type="search" class="form-control border-left-0 pl-0 shadow-none" placeholder="Search Inferno Documentation" aria-label="Search" aria-describedby="search-addon" id="search-input">
+          <input type="search" class="form-control border-left-0 pl-0 shadow-none"
+            placeholder="Search Inferno Documentation" aria-label="Search" aria-describedby="search-addon"
+            id="search-input">
         </div>
         <div id="search-results" class="search-results"></div>
       </div>
       <ul class="navbar-nav">
         <li class="nav-item">
-          <a class="nav-link{% if page.section == 'about' %} active{% endif %} align-middle" style="display: inline" href="{{site.baseurl}}/about/">About</a>
+          <a class="nav-link{% if page.section == 'about' %} active{% endif %} align-middle" style="display: inline"
```

Comment:
What's the purpose of "display: inline" on just the About link?

Reply:
Doesn't look like it's doing anything! Will remove, thanks